.github/workflows/pr_tests.yml

```diff
@@ -126,47 +126,25 @@ jobs:
       #     sudo apt-get update
       #     sudo apt-get install libavdevice-dev libavfilter-dev libopus-dev libvpx-dev pkg-config
 
-      - name: Install PyTorch Linux and MacOS
-        if: startsWith(runner.os, 'Windows') != true
+      - name: Adjust PyTorch versions
         run: |
-          ./scripts/pytorch_install.sh ${{ matrix.torch-version }}
-
-      - name: Install PyTorch Windows
-        if: startsWith(runner.os, 'Windows')
-        run: |
-          ./scripts/pytorch_install.ps1 ${{ matrix.torch-version }}
+          python .github/adjust_torch_versions.py ./requirements.txt ${{ matrix.torch-version }}
 
       - name: Install packages
         run: |
-          pip install -r requirements.txt --default-timeout=60
-          pip install -e . --default-timeout=60
+          pip install -r requirements.txt
+          pip install -e .
           pip freeze
+        timeout-minutes: 20
 
       - name: Run normal tests
         run: |
           pytest -m 'fast or slow' --cov syft --cov-fail-under 80 -n auto
 
-      - name: Install supported library packages Linux and MacOS - Ignore Errors
-        if: startsWith(runner.os, 'Windows') != true
-        run: |
-          cat requirements/supported_libs.txt | xargs -I {} -n 1 pip install {} --default-timeout=60 || true
-
-      - name: Install supported library packages Windows - Ignore Errors
-        if: startsWith(runner.os, 'Windows')
-        run: |
-          foreach ($i in cat requirements/supported_libs.txt) {
-            pip install $i --default-timeout=60
-          }
-
-      - name: Re-install PyTorch Linux and MacOS
-        if: startsWith(runner.os, 'Windows') != true
-        run: |
-          ./scripts/pytorch_install.sh ${{ matrix.torch-version }}
-
-      - name: Re-install PyTorch Windows
-        if: startsWith(runner.os, 'Windows')
+      - name: Install supported library packages
         run: |
-          ./scripts/pytorch_install.ps1 ${{ matrix.torch-version }}
+          pip isntall -r requirements/supported_libs.txt
```

Comment:
The reason we xargs the supported_libs is that they aren't available on all OSes or Python versions, so this allows them to silently fail to install and then those tests will simply not run.

Reply:
my bad, thx :]